crypto/CMakeLists.txt

```diff
@@ -147,7 +147,7 @@ if(HOST_ASM_MASM_X86_64)
 	add_definitions(-DWHIRLPOOL_ASM)
 	add_definitions(-DOPENSSL_CPUID_OBJ)
 	set(CRYPTO_SRC ${CRYPTO_SRC} ${ASM_X86_64_MASM_SRC})
-	set_property(SOURCE ${ASM_X86_64_MASM_SRC} PROPERTY LANGUAGE C)
+	set_property(SOURCE ${ASM_X86_64_MASM_SRC} PROPERTY LANGUAGE ASM_MASM)
```

Comment:
Files are getting picked up correctly by VS when setting the language to ASM_NASM